### DISCUSSION TITLE
Podman Desktop from Flathub configuration not saving

### DISCUSSION THREAD

**Discussion by hadrabap** [podman-desktop/podman-desktop#2116] [Q&A] [answered]
Hello friends,

Today I've upgraded Podman Desktop from Flathub to version 0.14.1.

The difference is obvious: there is no longer hidden dialog behind the main window. Great!

But I found another issue: whenever I change any configuration option, the settings is not stored anywhere on my disk. When re-launching Podman Desktop, the settings is in its defaults.

I tried to run Podman Desktop with `strace` with no luck. There is no single record which will suggest opening any configuration file. Is that feature even implemented on Linux/Flatpak?

Thanks in advance!

**Comment by benoitf** [comment]
hello @hadrabap the settings change should be in auto save mode, exiting focus of a field should save it ( before there was an update button)
Could you try changing focus of fields and monitor the configuration file to see if you see changes in it

About podman desktop/ flatpak we read the files through flatpak sandboxed environment. Do you always see the welcome page when you launch Podman Desktop ? It should be displayed only once and config stored as well at this time

**Reply by hadrabap** [reply]
Hello @benoitf!

Thanks for quick response!

The problem is that I see the welcome page every-time I launch Podman Desktop.

By the way: I've installed Podman Desktop into system, not user (the `--system` option). Maybe that's the problem? Other applications work well, however.

**Reply by hadrabap** [reply]
By the way: where should I look for the configuration file? I failed to find it. :-/

**Reply by benoitf** [reply]
https://github.com/containers/podman-desktop/blob/main/packages/main/src/plugin/configuration-registry.ts#L121-L124

The file that we check/read

**Comment by benoitf** [comment, marked as answer]
https://github.com/flathub/io.podman_desktop.PodmanDesktop/blob/master/io.podman_desktop.PodmanDesktop.yml#L15

We add 'home' access for flatpak

**Reply by hadrabap** [reply]
This is it!

Thanks a lot!

The upgrade process didn't add the host permission for some reason. Adding it manually like

```
flatpak override io.podman_desktop.PodmanDesktop --filesystem=host
```

as root (as Podman Desktop is installed into system) did the trick!

Thank you very much. Now I can see the JSON configuration file with appropriate values in it.
